src/util/createFilledBuffer.ts

```diff
@@ -0,0 +1,12 @@
+export function createFilledBuffer(sizeInMegaBytes: number): Buffer {
+  const characterBuffer = new Buffer('A');
```

Comment:
Using `new Buffer()` is [deprecated](https://nodejs.org/api/buffer.html#buffer_new_buffer_array).